modules/rosa-rhods.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * adding_service_cluster/rosa-available-services.adoc
+:_content-type: CONCEPT
+[id="rosa-rhods_{context}"]
+= Red Hat OpenShift Data Science
+
+{rhods} (RHODS) enables users to integrate data and AI and machine learning software to run end-to-end machine learning workflows. It provides a collection of notebook images with the tools and libraries required to develop and deploy data models. This allows data scientists to easily develop data models, integrate models into applications, and deploy applications using Red Hat OpenShift. RHODS is available as an add-on to Red Hat managed environments such as {osd} and {product-title} (ROSA).
```

Comment:
RHODS is not GAed yet. It is in Limited Availability. Should we have to add this info here ?

Reply:
@aloganat Do you know exactly when RHODS will be GA?

Reply:
Following up from Slack message, RHODS is in Limited Availability. Note added to clarify.